Agendas/20180314.md

```diff
@@ -11,7 +11,7 @@
     9. [Issue #84](https://github.com/oasis-tcs/sarif-spec/issues/84): "Enable localization for all message strings"
     10. [Issue #97](https://github.com/oasis-tcs/sarif-spec/issues/97): "file object's contents property"
     11. Opportunistic change: redefinition of `invocation.responseFiles` property
-    12. Opportunitic change: improvement to definition of `replacement` object
+    12. Opportunistic change: improvement to definition of `replacement` object
```

Comment:
I'd hope that changes like this are deemed `editorial` and acceptable for changing even in dated files. They make it much easier to read and understand what's going on.

That said, I can understand not wanting to make such changes.